DSharpPlus.Interactivity/Enums/ButtonPaginationBehavior.cs

```diff
@@ -0,0 +1,43 @@
+// This file is part of the DSharpPlus project.
+//
+// Copyright (c) 2015 Mike Santiago
+// Copyright (c) 2016-2021 DSharpPlus Contributors
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+namespace DSharpPlus.Interactivity.Enums
+{
+    /// <summary>
+    /// Represents options of how to handle pagination timing out.
+    /// </summary>
+    public enum ButtonPaginationBehavior
+    {
+        /// <summary>
+        /// The buttons should be disabled when pagination times out.
+        /// </summary>
+        Disable,
+        /// <summary>
+        /// The buttons should be left as is when pagination times out.
+        /// </summary>
+        Ignore,
+        /// <summary>
+        /// The entire message should be deleted when pagination times out.
+        /// </summary>
+        DeleteMessage
+    }
```

Comment:
Can we also have one that deletes the buttons instead of only disabling them?

Reply:
@yoshichulo Implemented.